.travis.yml

```diff
@@ -28,6 +29,11 @@ matrix:
       env: SYMFONY_VERSION='3.0.*'
     - php: 7.0
       env: DEPENDENCIES='dev'
+  fast_finish: true
```

Comment:
A question, on a side note: 5.3/5.4 are EOL, should they not be under an `allowed_failures` section?